src/components/problem/styledProblem.js

```diff
@@ -40,24 +88,37 @@ export const Subtitle = styled.p`
   font-weight: 400;
   line-height: 24px;
   letter-spacing: 0em;
-`;
 
-export const Highlight = styled.span`
-  color: #FF9820;
+  @media( max-width: 1023px ) {
+    max-width: 616px;
+    text-align: center;
+    margin: 0 auto;
+  }
 `;
 
-export const StatCell = styled.div`
+export const CircleCell = styled.div`
   background: #fff;
-  border-radius: 20px;
+  border-radius: 3px;
   box-shadow: 0px 4px 4px 0px #00000040;
   display: flex;
   flex-direction: column;
   gap: 30px;
   justify-content: center;
   align-items: center;
+
+  @media( max-width: 1023px ) {
```

Comment:
At 1024px the circle cell is supposed to be slightly larger than a grid cell.  This is pretty annoying from an implementation standpoint, but if we want to be 100% figma accurate, that's what the design says.
Cell= 314px
Circle = 347px

Reply:
This was one of those where I didn't think it could be done so I deviated. I think I figured it out. It's less flexible but looks more like the design now.